roles/zuul/templates/etc/zuul/zuul.conf

```diff
@@ -19,24 +19,38 @@ start = {{ zuul_gearman_server_start }}
 
 {% endfor %}
 
+{% if not zuul_zuul_v3 %}
 {% for name, details in zuul_sites.items() %}
 [site "{{ name }}"]
 {% for key, value in details.items() %}
 {{ key }} = {{ value }}
 {% endfor %}
 
 {% endfor %}
+{% endif %}
 
 [zuul]
+state_dir = /var/lib/zuul
+url_pattern = {{ zuul_logs_url }}/{build.parameters[LOG_PATH]}
+{% if zuul_zuul_v3 %}
+log_config = /etc/zuul/scheduler-logging.conf
+pidfile = /var/run/zuul-scheduler/zuul-sscheduler.pid
```

Comment:
damn, - i'm actually pretty sure this gets ignored when we don't use the daemon mode anyway.